lib/logstash/inputs/rabbitmq/march_hare.rb

```diff
@@ -112,6 +112,9 @@ def consume
       @consumer = @q.build_consumer(:block => true) do |metadata, data|
         @codec.decode(data) do |event|
           decorate(event)
+	  metadata.headers.each do |header, value|
+            event[header] = value.respond_to?('toString') ? value.toString : value 
+         end if @include_headers
```

Comment:
Hey @4n3w any updates here on the toString stuff? I assume this is java arrays not being properly typecast. I think the proper behavior here is to cast any native Java values to ruby values, e.g. ArrayList -> List, java.lang.String.